MN/Algoritmi/Ecuatii-Neliniare/Secant_Method.m

```diff
@@ -0,0 +1,36 @@
+function [x] = Secant_Method(xn_2, xn_1, tol, p, max_iter)
```

Comment:
n-as pune '_' in numele functiei si as numi explicit primii doi termeni x0 si x1. Vrem si numarul de pasi in care ajungem la solutie. De asemenea, nu mai lucram doar cu polinoame, ci vrem orice tip de functie transmisa prin referinta. :octocat:
```matlab
function [x i] = SecantMethod(f, x0, x1, tol, max_iter)
```